config/src/keyassignment.rs

```diff
@@ -17,6 +17,8 @@ use wezterm_term::SemanticType;
 pub struct LauncherActionArgs {
     pub flags: LauncherFlags,
     pub title: Option<String>,
+    pub description: Option<String>,
```

Comment:
Naming is the hardest part of this.  I think I regret using `title` for the tab title.
I think description is a bit too vague.

Rather than description, I think that maybe something like `help_text` is a bit more indicative of what this option does.

And then `fuzzy_help_text` for the fuzzy mode?